### PR TITLE
update formula to Teresa v0.12.0

### PR DESCRIPTION
**Pull request by drgarcia1986**

### CODE REVIEW

File: Formula/teresa-cli.rb [hunk 1]
```diff
@@ -2,9 +2,9 @@ class TeresaCli < Formula
   desc "Teresa client"
   homepage "https://github.com/luizalabs/teresa"
   url "https://github.com/luizalabs/teresa/releases/download/v0.11.0/teresa-darwin-amd64"
```
Comment (rafaelgotts) [inline Formula/teresa-cli.rb]:
```diff
- "https://github.com/luizalabs/teresa/releases/download/v0.11.0/teresa-darwin-amd64"
+ "https://github.com/luizalabs/teresa/releases/download/v0.12.0/teresa-darwin-amd64"
```

**Review by yagonobre — APPROVED** [review]

File: Formula/teresa-cli.rb (hunk 1 above)
Comment (drgarcia1986) [reply Formula/teresa-cli.rb]: 🤦‍♂️

**Comment by drgarcia1986** [comment]
@rafaelgotts updated

**Review by rafaelgotts — APPROVED** [review]
👏

**Pull request merged**